spinn_front_end_common/interface/interface_functions/on_chip_router_table_compression/compression.py

```diff
@@ -36,7 +36,7 @@
 def mundy_on_chip_router_compression(
         routing_tables, transceiver, machine, app_id,
         system_provenance_folder, compress_only_when_needed=True,
-        compress_as_much_as_possible=False):
+        compress_as_much_as_possible=False, write_compressor_iobuf=False):
```

Comment:
this isnt really sensible. its never called directly, and the 2 places that do call it take the write_compressor_iobuf as a required parameter. so having a default value here makes no sense.

Reply:
cleaned up for ya